src/intelligence_layer/connectors/argilla/argilla_wrapper_client.py

```diff
@@ -215,31 +225,48 @@ def records(self, dataset_id: str) -> Iterable[Record]:
             for record in remote_dataset.records
         )
 
-    def _create_evaluation(self, record_id: str, data: dict[str, Any]) -> None:
-        api_url = os.environ["ARGILLA_API_URL"]
-        if not api_url.endswith("/"):
-            api_url = api_url + "/"
-        rg.active_client().http_client.post(
-            f"{api_url}api/v1/records/{record_id}/responses",
-            json={
-                "status": "submitted",
-                "values": {
-                    question_name: {"value": response_value}
-                    for question_name, response_value in data.items()
-                },
-            },
-        )
+    def _create_evaluation(
+        self, dataset_id: str, record_id: str, data: dict[str, Any]
+    ) -> None:
+        dataset = self._dataset_from_id(dataset_id=dataset_id)
+        if dataset is None:
+            raise ValueError("dataset does not exst")
+        records = dataset.records
+
+        user_id = self.client.me.id
+        if user_id is None:
+            raise ValueError("user_id is not a UUID")
+
+        # argilla currently does not allow to retrieve a record by id
```

Comment:
we could change the function to take two lists for records and data, so we only need to loop once, but its ok like this since its only used for a single test and private anyway imo

Reply:
As discussed, added a comment but since we basically don't use it, won't change this now